static/app/components/organizations/pageFilterBar.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled';
+
+import {StyledButton} from 'sentry/components/button';
+
+const PageFilterBar = styled('div')`
+  display: flex;
+  border: solid 1px ${p => p.theme.border};
+  border-radius: ${p => p.theme.borderRadius};
+
+  & ${StyledButton} {
```

Comment:
is this for the pin button/new project button?

Reply:
oh shoot good catch… fixed in c6dc8f83b44f60d387e050ecf62c67102bff2494